File: A1/A1-022/A1-022.cpp
#include<iostream>
using namespace std;

int main()
{
    // This makes std::cin and std::cout faster
    ios_base::sync_with_stdio(false); cin.tie(0);

    // Input
    int day, month;
    cin >> day >> month;
    // Output
    pair<int,int> date = {month, day};
    if(date <= make_pair(1, 19)) { cout << "capricorn"; }
    else if(date <= make_pair(2, 18)) { cout << "aquarius"; }
    else if(date <= make_pair(3, 20)) { cout << "pisces"; }
    else if(date <= make_pair(4, 19)) { cout << "aries"; }
    else if(date <= make_pair(5, 20)) { cout << "taurus"; }
    else if(date <= make_pair(6, 21)) { cout << "gemini"; }
    else if(date <= make_pair(7, 22)) { cout << "cancer"; }
    else if(date <= make_pair(8, 22)) { cout << "leo"; }
    else if(date <= make_pair(9, 22)) { cout << "virgo"; }
    else if(date <= make_pair(10, 23)) { cout << "libra"; }
    else if(date <= make_pair(11, 21)) { cout << "scorpio"; }
    else if(date <= make_pair(12, 21)) { cout << "sagittarius"; }
    else { cout << "capricorn"; }
    return 0;
}

File: A1/A1-022/A1-022.py
# --------------------------------------------------
# File Name : A1-022.py
# Problem   : Zodiac
# Author    : Worralop Srichainont
# Date      : 2026-02-08
# --------------------------------------------------

# List of zodiac name and its date range
ZODIACS_INFO = (
    ([12, 22], [12, 31], "capricorn"),
    ([1, 1], [1, 19], "capricorn"),
    ([1, 20], [2, 18], "aquarius"),
    ([2, 19], [3, 20], "pisces"),
    ([3, 21], [4, 21], "aries"),
    ([4, 22], [5, 20], "taurus"),
    ([5, 21], [6, 21], "gemini"),
    ([6, 22], [7, 22], "cancer"),
    ([7, 23], [8, 22], "leo"),
    ([8, 23], [9, 22], "virgo"),
    ([9, 23], [10, 23], "libra"),
    ([10, 24], [11, 21], "scorpio"),
    ([11, 22], [12, 21], "sagittarius"),
)

# Input date
day = int(input())
month = int(input())
date = [month, day]

# Find a zodiac for the input date
ans = ""
for start_date, stop_date, zodiac_name in ZODIACS_INFO:
    if start_date <= date <= stop_date:
        ans = zodiac_name
        break

# Output
print(ans)
